Newsletters: add 334, the 2024 year-in-review special

harding opened the pull request:
This is still raw and some sections need additional details, so I'll be back with additional commits in a few hours.  Sorry for the lateness.

jirijakes commented on _posts/en/newsletters/2024-12-20-newsletter.md:
```suggestion
Friday publication schedule on January 3rd.*
```

LarryRuane commented on _posts/en/newsletters/2024-12-20-newsletter.md:
```suggestion
development this year, the rough equivalent of a 500-page book.
```
According to google, the average nonfiction book has about 233 words per page.

harding replied:
That's pretty sparse.  I think I'm going to keep the assumption of about 350 words per page, since I believe that's what we've used for comparisons in the past and I think it roughly corresponds to technical books like from O'Reilly (e.g., Mastering Bitcoin 3rd ed is 130,000 words / 400 pages = 325 words per page).

harding commented:
Added a commit with the remaining main content (except the list of releases).  I'll be back later to begin addressing feedback, clean things up, and add that list.  Thanks for the early reviews @jirijakes and @LarryRuane !

bitschmidty commented on _posts/en/newsletters/2024-12-20-newsletter.md:
```suggestion
  extensible, added support for [BIP129][] multisig wallet import,
```

bitschmidty commented on _posts/en/newsletters/2024-12-20-newsletter.md:
```suggestion
  delivery of [offer][topic offers] invoices, and fixed a fee
```

bitschmidty commented on _posts/en/newsletters/2024-12-20-newsletter.md:
```suggestion
  making payments that use advanced route selection; paying and
```

bitschmidty reviewed (commented):
@harding incredible as always! 🧡 

Added an HRF contribution blurb

harding commented on _posts/en/newsletters/2024-12-20-newsletter.md:
It's weird with it and weird without it.  I'm gonna leave it in but no strong preference either way.

harding commented:
Just read through the whole document and pushed my own edits.  I'm hoping on a flight soon but I'll be back in about 12 hours to address any final feedback.

Thanks so much for the feedback so far @LarryRuane @jirijakes @bitschmidty !

harding commented:
@bitschmidty oh, maybe you want to add something to the Optech section about all the translations this year.  You and the translators handle that so efficiently in the background that I forget about it sometimes.

LarryRuane commented on _posts/en/newsletters/2024-12-20-newsletter.md:
```suggestion
  improved plugin flexibility and began migrating all altcoin support to
```

LarryRuane commented on _posts/en/newsletters/2024-12-20-newsletter.md:
```suggestion
  script opcode behavior".  The release dropped support for `annexdatacarrier`
```

harding replied:
Leaving as-is.  Passive is more efficient here.

LarryRuane commented on _posts/en/newsletters/2024-12-20-newsletter.md:
```suggestion
  [full-RBF][topic rbf].  The release added default parameters for [assumeUTXO][topic
  assumeutxo], allowing use of the `loadtxoutset` RPC
```

harding replied:
Leaving as-is.

harding commented on _posts/en/newsletters/2024-12-20-newsletter.md:
Keeping as-is but I'll remove the "at the time" from the next sentence.

harding commented on _posts/en/newsletters/2024-12-20-newsletter.md:
Leaving as-is.  I want to make it clear that this is a judgment call we're making unilaterally.  There's no objective standard for what constitutes "responsible" disclosure, so we have to make a guess (often based on limited information).

harding commented on _posts/en/newsletters/2024-12-20-newsletter.md:
Leaving as-is.  I'm not familiar with this rule.

harding commented on _posts/en/newsletters/2024-12-20-newsletter.md:
Leaving as-is.  I think a short sentence improves the flow here.

harding commented on _posts/en/newsletters/2024-12-20-newsletter.md:
Leaving as-is.  January doesn't own LNHANCE and it's the proposal, not the announcement, that includes previous proposals.

harding commented on _posts/en/newsletters/2024-12-20-newsletter.md:
Leaving as-is.

bitschmidty reviewed (approved):
ACK

Skimmed edits. Added updates to Optech section (included last podcast in the totals, added translations callout)

bitschmidty commented:
Squashed and merged. Thank you @harding for authoring the entire year along with all of our special and notable code segment contributors, translators, and reviewers! 🚀
